src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 #MySQL DB Config
 db.driver=com.mysql.cj.jdbc.Driver
-db.url=jdbc:mysql://host:port/db_name
-db.username=db_user
-db.password=db_password
+db.url=jdbc:mysql://localhost:3306/inventory_db?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
+db.username=root
+db.password=1234
```

Comment:
Don't push it, please :-)

Reply:
ok ok :D

Reply:
use approach with ENV vars which I showed on Wednesday